hydra-cardano-api/hydra-cardano-api.cabal

```diff
@@ -92,6 +92,7 @@ library
     , cardano-crypto-class    >=2.1.1  && <2.2
     , cardano-ledger-allegra  >=1.2.1  && <1.3
     , cardano-ledger-alonzo   >=1.4    && <1.5
+    , cardano-ledger-api      >=1.5    && <1.6
```

Comment:
Why do we need this change here?

Reply:
It's the intended package to use when dealing with ledger types nowadays. I was missing at least one of the lenses, can't remember which.

Reply:
`setMinCoinTxOut` is only defined in `cardano-ledger-api`